backend/inspirehep/curation/views.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from flask import Blueprint, abort
+from invenio_db import db
+from webargs import fields
+from webargs.flaskparser import FlaskParser
+
+from inspirehep.accounts.decorators import login_required_with_roles
+from inspirehep.accounts.roles import Roles
+from inspirehep.records.api import LiteratureRecord
+from inspirehep.serializers import jsonify
+
+blueprint = Blueprint("inspirehep_curation", __name__, url_prefix="/curation")
+parser = FlaskParser()
+
+
+@parser.error_handler
+def handle_error(error, req, schema, error_status_code, error_headers):
+    message = f"""Incorrect input type for fields: {" ".join(error.field_names)}"""
+    abort(400, message)
+
+
+@blueprint.route("/literature/<int:pid_value>/keywords", methods=["PUT"])
+@login_required_with_roles([Roles.cataloger.value])
+@parser.use_kwargs(
```

Comment:
Minor: I think using `use_args` is a little bit more clear in this case, as you don't have `required` fields, so you need to use `**kwargs` (instead of `args`).